kubernetes-manifests/client.deployment.yaml

```diff
@@ -66,6 +66,12 @@ spec:
               value: "true"
             - name: GRPC_EXPERIMENTAL_XDS_ENABLE_OVERRIDE_HOST
               value: "true"
+            % if enable_dualstack:
```

Comment:
For this variable to be available in the template, we need to set it in `KubernetesClientRunner`. Add it to `ClientDeploymentArgs`, see https://github.com/grpc/psm-interop/blob/fc57159059a70534fef4136a15659eca9e140869/framework/test_app/runners/k8s/k8s_xds_client_runner.py#L29-L33

Reply:
Done